src/main/java/org/motechproject/newebodac/mapper/RoleMapper.java

```diff
@@ -0,0 +1,17 @@
+package org.motechproject.newebodac.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.ReportingPolicy;
+import org.mapstruct.factory.Mappers;
+import org.motechproject.newebodac.domain.security.UserRole;
+import org.motechproject.newebodac.dto.RoleDto;
+
+@Mapper(uses = { UuidMapper.class },
+    unmappedTargetPolicy = ReportingPolicy.IGNORE)
+public interface RoleMapper extends EntityMapper<RoleDto, UserRole> {
+
+  RoleMapper INSTANCE = Mappers.getMapper(RoleMapper.class);
+
```

Comment:
you will need to add new methods here to map UUID to UserPermission and other way around